src/test/scala/uk/gov/hmrc/test/ui/pages/section6/ContainerListPage.scala

```diff
@@ -42,4 +44,7 @@ object ContainerListPage extends BasePage {
   // Yes => fillPage(yes)
 
   override def fillPage(values: String*): Unit = selectYesOrNoRadio(values(yesNo))
+
+  def removeContainer(index: Int): Unit = clickByCssSelector(s"#containers-row$index-remove_button > a")
```

Comment:
WDYT of renaming this to `selectContainerToRemove` instead of `removeContainer`? Because this action does not mean to actually remove a container.

Reply:
Done